### ISSUE TITLE
pah8001_libs 

### ISSUE THREAD

**Issue by fanoush** [blockswearables/blocks-module-heart-rate#3]
Hello, your repository includes module https://github.com/pixartinc/pah8001_libs but that is (no longer?) available. Can you help with getting that library?

I am working on [custom firmware for DS-D6 fitnes tracker](https://github.com/fanoush/ds-d6) with this HR sensor and looks like  Ineed this library to compute heart rate from raw values. Thank you.

**Comment by 5361** [comment]
Indeed, getting access to this library would be of tremendous help to process the data retrieved by a PAH8001EI-2G HR sensor. Thx
